prometheus_api_client/prometheus_connect.py

```diff
@@ -389,17 +393,25 @@ def get_metric_aggregation(self, query: str, operations: list, params: dict = No
             _LOGGER.debug("No operations found to perform")
             return None
         aggregated_values = {}
-        data = self.custom_query(query, params)
-        values = []
+        query_values = []
+        if start_time is not None and end_time is not None:
+            data = self.custom_query_range(query=query, params=params,
+                                           start_time=start_time, end_time=end_time, step=step)
```

Comment:
you think we should keep a default value for `step` that is not `None`? maybe something like 10 seconds?

Reply:
@amitsagtani97